TOOLS/dylib-unhell.py

```diff
@@ -80,15 +80,17 @@ def process_libraries(libs_dict, binary, processed = []):
             if p in libs_dict[src]:
                 install_name_tool_change(p, lib_name(p), dst)
 
-    process_libraries(libs_dict, binary, ls)
+    return ls
 
 def main():
     binary = os.path.abspath(sys.argv[1])
     if not os.path.exists(lib_path(binary)):
         os.makedirs(lib_path(binary))
     libs = libraries(binary)
-    print(libs)
```

Comment:
just to confirm that it is fine to remove this. this script is only called from [here](https://github.com/mpv-player/mpv/blob/master/TOOLS/osxbundle.py#L83), which doesn't return the output anyway. so i just believe this is some debug remnant from when this was split from `osxbundle.py`.

Reply:
yeah, i think this was just debugging. doubly so if compile works without it.